woocommerce/wc-api-dev
# Add a fix for keeping the site title and email footer in sync.

@ryelle @allendav This should help with https://github.com/Automattic/wp-calypso/issues/16445.

This really makes me wish we had an additional plugin for things like this (since it is not something we want in core as-is), but as I mentioned on the call, I think this will be fine for now as long as we keep these things in hotfixes. And maybe add to the GitHub README in a separate PR.

I haven't bumped the version here since @ryelle already is. These can go in the same release.

To Test:
* Apply this patch
* Change your site title
* Visit `/store/order/{domain}/{orderID}` and fulfill an order, giving it a tracking number (make anything up) and checking the "send to customer" box.
* Make sure the subject and footer have your new site title, without having to go to wp-admin to update anything
